kahuna/public/js/search/query.html

```diff
@@ -55,6 +56,17 @@
             </li>
 
             <li class="search__filter-item">
+                <label class="search__filter">
+                    <input type="checkbox"
+                           ng:model="searchQuery.ordering.orderBy"
+                           ng:false-value="undefined"
+                           ng:true-value="'oldest'"
+                           ng:checked="searchQuery.ordering.orderBy === 'oldest'" />
+		    Oldest first
```

Comment:
In that case it could be to the right of the date filter, which is also a filter?

I wonder if a dropdown would be clearer than a checkbox, if we want to avoid the space-intensive styled radio we had before?